src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Offender.kt

```diff
@@ -80,6 +80,9 @@ data class Offender(
   @OneToMany(mappedBy = "offender", cascade = [CascadeType.ALL], fetch = FetchType.LAZY, orphanRemoval = true)
   val bookings: MutableList<OffenderBooking> = mutableListOf(),
 
+  @OneToMany(mappedBy = "rootOffender", fetch = FetchType.LAZY)
+  private val allBookings: MutableList<OffenderBooking> = mutableListOf(),
+
```

Comment:
We still have the `@Deprecated` bookings above - I don't think we need to leave deprecated code lying around for ourselves to unpick later when we've forgotten what's going on. Can we get rid of this now?

Reply:
I left it in because it is used quite widely in the tests in various ways to setup data etc and the volume of changes were getting out of hand. Maybe a follow-up PR

Reply:
OK cool. But I think it needs to go sooner rather than later if possible.